src/codegen/mod.rs

```diff
@@ -3871,4 +3925,28 @@ mod utils {
 
         args
     }
+
+    pub fn fnsig_block(
+        ctx: &BindgenContext,
+        sig: &FunctionSig,
+    ) -> quote::Tokens {
+        let args = sig.argument_types().iter().map(|&(_, ty)| {
```

Comment:
I think `ty.to_rust_ty_or_opaque(ctx, &())` would do just fine here, no need to `try_to_rust_ty`, unless you have a test-case in which it makes a difference.